pkg/apis/eventsource/v1alpha1/types.go

```diff
@@ -379,26 +379,19 @@ type GitlabEventSource struct {
 	Webhook *webhook.Context `json:"webhook" protobuf:"bytes,1,name=webhook"`
 	// ProjectId is the id of project for which integration needs to setup
 	ProjectId string `json:"projectId" protobuf:"bytes,2,name=projectId"`
-	// Event is a gitlab event to listen to.
+	// Events are gitlab event to listen to.
```

Comment:
Since you are changing the types, can you update `ProjectId` above to `ProjectID`?

Reply:
Sure